app/controllers/users_controller.rb

```diff
@@ -1,13 +1,12 @@
 class UsersController < ApplicationController
-  has_filters %w{proposals debates budget_investments comments}, only: :show
+  has_filters %w{proposals debates budget_investments comments follows}, only: :show
 
   load_and_authorize_resource
   helper_method :author?
-  helper_method :author_or_admin?
```

Comment:
👍  you're right, I did a project search and no use, although sometimes looking for the method string is not enough if you do compose de method name with a string interpolation like `send("#{this}_or_#{that}")` but that's not the case I guess based on the specs passing ^_^

Nice catch & cleanup 👏 😄